README.md

```diff
@@ -1 +1,30 @@
-ArmA.Studio is a IDE developed specially for the ArmA game.
\ No newline at end of file
+## Arma.Studio IDE
+
+ArmA.Studio is a IDE developed specially for the ArmA game.
+
+ArmA.Studio uses the [ArmaDebugEngine](https://github.com/dedmen/ArmaDebugEngine).
+
+### Links
+
+[Website (Download)](https://x39.io/projects?project=ArmA.Studio)
+
+[Bi-Forum Thread](https://forums.bistudio.com/forums/topic/203020-armastudio-the-open-source-community-made-ide-for-arma/) / [Armaholic](http://www.armaholic.com/page.php?id=32550)
+
```

Comment:
Final result looks like
`Bi-Forum Thread / Armaholic Armaworld (German)` Need to add more Newlines if it should be on seperate lines. Or add another `/`